frontend/src/components/networkpolicy/Details.tsx

```diff
@@ -0,0 +1,190 @@
+                {
+                  name: t('ipBlock'),
+                  value: item.from.map(from => {
+                    if (!from.ipBlock) {
+                      return <></>;
+                    }
+                    const { cidr, except = [] } = from.ipBlock || {};
+                    if (!cidr) {
+                      return <></>;
+                    }
+                    if (cidr && except.length === 0) {
+                      return <>{`cidr: ${cidr}`}</>;
+                    }
+                    return <>{`cidr: ${cidr}, except: ${except.join(', ')}`}</>;
+                  }),
+                },
+                {
+                  name: t('namespaceSelector'),
+                  value: item.from.map(from => {
+                    if (!from.namespaceSelector) {
+                      return <></>;
+                    }
+                    const { matchLabels = {}, matchExpressions = [] } =
+                      from.namespaceSelector || {};
+                    return prepareMatchLabelsAndExpressions(matchLabels, matchExpressions);
+                  }),
+                },
+                {
+                  name: t('podSelector'),
+                  value: item.from.map(from => {
+                    if (!from.podSelector) {
+                      return <></>;
+                    }
+                    const { matchLabels = {}, matchExpressions = [] } = from.podSelector || {};
+                    return prepareMatchLabelsAndExpressions(matchLabels, matchExpressions);
+                  }),
+                },
+              ]}
+            />
+          </SectionBox>
+        ))}
+      </>
+    );
+  }
+
+  function Egress(props: { egress: NetworkPolicyEgressRule[] }) {
+    const { egress } = props;
+    if (!egress || egress.length === 0) {
+      return <></>;
+    }
+    return (
+      <>
+        {egress.map((item: NetworkPolicyEgressRule) => (
+          <SectionBox title={t('Egress')}>
+            <NameValueTable
+              rows={[
+                {
+                  name: t('Ports'),
+                  value: item.ports.map((port: NetworkPolicyPort) => (
+                    <Box>
+                      {port.protocol}:{port.port}
+                    </Box>
+                  )),
+                },
+                {
+                  name: t('To'),
+                  value: '',
+                },
+                {
+                  name: t('ipBlock'),
+                  value: item.to.map(to => {
+                    if (!to.ipBlock) {
+                      return <></>;
+                    }
+                    const { cidr, except = [] } = to.ipBlock || {};
+                    if (!cidr) {
+                      return <></>;
+                    }
+                    if (cidr && except.length === 0) {
+                      return <>{`cidr: ${cidr}`}</>;
+                    }
+                    return <>{`cidr: ${cidr}, except: ${except.join(', ')}`}</>;
```

Comment:
Please make this translatable (for the `except`)

Reply:
updated